docs/index.md~

```diff
@@ -33,7 +33,7 @@ If you would like to contribute a feature then fork the master branch (fork the
 ```
 nosetests -s -v tests/
 ```
-Let us know what you want to do just in case we're already working on an implementation of something similar. This way we can avoid any needless duplication of effort. Also, please don't forget to add tests for any new functions. 
```

Comment:
@jmschrei Is this whole file redundant with the index.rst file and should it be removed?